docs/tools/terraform/reference/cookbook/postgres-clickhouse-integration-recipe.rst

```diff
@@ -0,0 +1,160 @@
+    project       = var.project_name
+    service_name  = aiven_pg.postgres.service_name
+    database_name = "sensors"
+  }
+
+  // ClickHouse service based in the same region
+  resource "aiven_clickhouse" "clickhouse" {
+    project                 = var.project_name
+    service_name            = "clickhouse-gcp-us"
+    cloud_name              = "google-us-east-4"
+    plan                    = "startup-4"
+    maintenance_window_dow  = "monday"
+    maintenance_window_time = "10:00:00"
+  }
+
+  // Sample ClickHouse database that can be used to write and process raw data
+  resource "aiven_clickhouse_database" "iot_measurements" {
+    project      = var.project_name
+    service_name = aiven_clickhouse.clickhouse.service_name
+    name         = "iot_measurements"
+  }
+
+  // ClickHouse service integration for the PostgreSQL service as a source
+  resource "aiven_service_integration" "clickhouse_postgres_source" {
+    project                  = var.project_name
+    integration_type         = "clickhouse_postgresql"
+    source_service_name      = aiven_pg.postgres.service_name
+    destination_service_name = aiven_clickhouse.clickhouse.service_name
+  }
+
+Execute the Terraform files
+'''''''''''''''''''''''''''
+
+.. dropdown:: Expand to check out how to execute the Terraform files.
+
+  1. Run the following command:
+
+    .. code-block:: shell
+
+       terraform init
+  
+  The ``init`` command performs initialization operations to prepare the working directory for use with Terraform. For this recipe, ``init`` automatically finds, downloads, and installs the necessary Aiven Terraform Provider plugins.
+
+  2. Run the following command:
+
+    .. code-block:: bash
+
+       terraform plan -var-file=var-values.tfvars
+  
+  The ``plan`` command creates an execution plan and shows the resources to be created (or modified). This command doesn't actually create any resources but gives you a heads-up on what's going to happen next.
+
+  3. If the output of ``terraform plan`` looks as expected, run the following command:
+
+    .. code-block:: bash
+
+       terraform apply -var-file=var-values.tfvars
+  
+  The ``terraform apply`` command creates (or modifies) your infrastructure resources.
+
+Check out the results
+---------------------
+
+* Resource ``aiven_clickhouse`` creates an Aiven for ClickHouse service with the project name, the cloud name (provider, region, zone), the Aiven service plan, and the service name as specified in the ``services.tf`` file.
+* ``aiven_clickhouse_database`` resource creates a database that can be used to store high-throughput measurement data and to create new tables and views to process this data.
+* ``aiven_pg`` resource creates an Aiven for PostgreSQL service.
+* * ``aiven_pg_database`` resource creates the ``sensors`` database.
+* ``aiven_service_integration`` resource creates the integration between the Aiven for PostgreSQL and Aiven for ClickHouse services.
```

Comment:
This will result in the creation of a ``service_postgres-gcp-us_sensor_dims_public`` database in ClickHouse allowing the users to access the ``sensor_dims`` database for the ``postgres-gcp-us`` service.
It might be worth to point it out as the name might not be self-evident.